src/composables/useRunning/useRunning.ts

```diff
@@ -1,5 +1,6 @@
 import { ref } from 'vue';
 import { Vigad } from '../../proc/Vigad';
+import useNotificationSystem from '../useNotificationSystem/useNotificationSystem';
```

Comment:
i my opinion it would be better to use path's using @, they are then relativ to the src directory and won't change much

import useNotificationSystem from '@/composables/useNotificationSystem/useNotificationSystem';